modules/ROOT/pages/design-common-errors-10.adoc

```diff
@@ -0,0 +1,325 @@
+= Common Errors in Conforming to RAML 1.0
+
+When testing its new RAML parser (which was released on October 24, 2018), MuleSoft identified eight common errors in API specifications written in RAML 1.0. The errors are explained with examples below. They are not listed in any particular order, such as frequency of occurrence in the sample of API specifications that MuleSoft tested.
+
+If you open in the code editor a published API specification that is written in RAML 0.8 and you see error indicators, this topic might help you to resolve most of the errors. If you see errors that this topic does not mention, refer to the specification for RAML 1.0, which you can read on https://raml.org/[raml.org].
+
+== Common Error 1
+// APIMF-834
+*_Appending references with hash symbols to filenames in `!include` statements_*
+
+A filename cannot be followed by a hash symbol and a reference to a location within the named file. In this example snippet, `IncrementType.raml#increment` is not a valid link.
+
+----
+#%RAML 1.0 DataType
+
+type: object
+properties:
+  startValue: integer
+  endValue: integer
+  exclusiveEndValue: boolean
+  range:
+    type: array
+    items: !include IncrementType.raml#increment
+
+----
+
+If your specification contains an error of this type, but you meant to write a comment, place an empty space before the "#" symbol. If you meant to reference an element that is in the file, such references are not allowed. References to inner elements are valid only for XSD and JSON schemas.
+
+
+== Common Error 2
+// APIMF-849
+*_Not correctly using curly braces and brackets in JSON examples_*
+
+There are many ways to misuse curly braces and brackets. This snippet illustrates one of them. An array of groups of JSON key/value pairs is improperly enclosed in a pair of curly braces.
+
+----
+#%RAML 1.0
+title: ExampleRAML
+...
+/rooms:
+  displayName: rooms
+  get:
+    description: get all rooms
+    responses:
+      200:
+        body:
+          application/json:
+            example:
+             {
+               [{
+                "Name": "Superior King",
+                "Number": "201",
+                "Property": "SE030",
+                "Status": "Clean"
+                },
+                {
+                "Name": "Junior Suite",
+                "Number": "202",
+                "Property": "NO131",
+                "Status": "Clean"
+                }]
+              }
+----
```

Comment:
If you want to use a json example, the body text should be expressed as a multiline text (starting with '|' char).

Reply:
So, line 48 should be `    example:   | ` ?

Reply:
Fixed.